app/assets/javascripts/application.js

```diff
@@ -70,41 +71,45 @@ $(document).ready(function() {
 
   // For DataTables and Bootstrap
   $('.datatable').dataTable({
-    "sDom": "<'row'<'span4'l><'span5'f>r>t<'row'<'span3'i><'span6'p>>",
-    "sPaginationType": "bootstrap",
-    "sScrollX": "100%",
-    "aoColumnDefs": [
-          { "bSortable": false, "aTargets": [ "no_sort" ] }
-        ]
+    "pagingType": "full_numbers",
+    "scrollX": false,
+    "columnDefs": [
+      { "orderable": false, "targets": [ "no_sort" ] }
+    ]
   });
 
-  wideDataTables = $('.datatable-wide').dataTable({
-    "sDom": "<'row'<'span5'l><'span7'f>r>t<'row'<'span5'i><'span7'p>>",
-    "sPaginationType": "bootstrap",
-    "sScrollX": "100%",
-    "aoColumnDefs": [
-          { "bSortable": false, "aTargets": [ "no_sort" ] }
-        ]
+  var wideDataTables = $('.datatable-wide').dataTable({
+    "pagingType": "full_numbers",
+    "scrollX": false,
+    "columnDefs": [
+      { "orderable": false, "targets": [ "no_sort" ] }
+    ]
   });
 
   // ### REPORTS JS ### //
 
   $('.report_table').dataTable({
-    "sDom": "<'row'<'span3'l>fr>t<'row'<'span3'i><p>>",
-    "sPaginationType": "bootstrap",
-    "iDisplayLength" : 25,
-    "aLengthMenu": [[25, 50, 100, -1], [25, 50, 100, "All"]],
-    "aoColumnDefs": [{ "bSortable": false, "aTargets": [ "no_sort" ] }]
+    "pagingType": "full_numbers",
+    "pageLength": 25,
+    "lengthMenu": [[25, 50, 100, -1], [25, 50, 100, "All"]],
+    "columnDefs": [{ "orderable": false, "targets": [ "no_sort" ] }]
   });
 
   // For fading out flash notices
   $(".alert .close").click( function() {
-       $(this).parent().addClass("fade");
+    $(this).parent().addClass("fade");
   });
 
   // make the sidebar follow you down the page
   if ($(window).width() > 767) {
-    $("#sidebarbottom").sticky({topSpacing: 50, bottomSpacing: 200});
+    $("#sidebarbottom").affix({
+      offset: {
+        top: 0,
+        bottom: function() {
```

Comment:
Nope, this happens even with a fixed integer bottom offset. Affix seems to struggle with calculating offsets – when looking at the inspector, I can see that when it freaks out it's toggling back and forth over and over between the two Affix states `.affix` and .`affix-bottom`.